src/foam/nanos/bench/SigningBenchmark.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.bench;
+
+import foam.core.X;
+import foam.dao.DAO;
+import foam.dao.ListSink;
+import foam.dao.Sink;
+import net.nanopay.tx.model.Transaction;
```

Comment:
This adds nanopay dependencies to foam2.  Can we add this class to the NANOPAY repo instead?  Standalone nanos is broken as a result.

Reply:
Yeah, my mistake.